lib/web/apiserver.go

```diff
@@ -498,7 +498,12 @@ func (h *Handler) getUserContext(w http.ResponseWriter, r *http.Request, p httpr
 		return nil, trace.Wrap(err)
 	}
 
-	userContext, err := ui.NewUserContext(user, roleset)
+	pingResponse, err := clt.Ping(r.Context())
```

Comment:
Is this new API method? Did it exist in 6.0 ?

Reply:
The method existed since last year, but sending back the `features` was added in `6.1`

Reply:
@kimlisa can we call this method once upon `NewHandler` initialization? This way we do not need to do make an extra call each time Web UI loads.

Also, since it was added recently, will it break if ping method does not have this struct?